React/Base/RCTUIKit.h

```diff
@@ -369,7 +372,7 @@ CGPathRef UIBezierPathCreateCGPathRef(UIBezierPath *path);
 
 #define RCTPlatformWindow NSWindow
 
-@interface RCTUIView : NSView
+@interface RCTUIView : RCTPlatformView
```

Comment:
Curious, what break did this fix? Or was it more "why not?"

Reply:
It seemed cleaner to me to not have multiple refs to `NSView`